### DISCUSSION TITLE
Welcome to filachat Discussions!

### DISCUSSION THREAD

**Discussion by 199ocero** [199ocero/filachat#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by njugunamwangi** [comment]
Is the navigation label customizable? From Filachat to something else

**Reply by njugunamwangi** [reply]
Found it

**Reply by 199ocero** [reply]
Glad you found it!

**Reply by njugunamwangi** [reply]
How about customizing the model label(page title)
